strings/valentines.json

```diff
@@ -2,24 +2,23 @@
     "valentines": [
         "Roses are red / Violets are good / One day while Andy...",
         "My love for you is like the singularity. It cannot be contained.",
-        "Will you be my lusty xenomorph maid?",
+        "You make me lustier than a xenomorph maid.",
         "We go together like the clown and the external airlock.",
         "Roses are red / Liches are wizards / I love you more than a whole squad of lizards.",
         "Be my valentine. Law 2.",
         "You must be a mime, because you leave me speechless.",
 		"I love you like Ian loves the HoP.",
 		"You're hotter than a plasma fire in toxins.",
 		"We must have a wizard, because your face summoned magic.",
-		"Are you a rogue atmos tech? Because you're taking my breath away.",
+		"Are you a hull breach? Because you're taking my breath away.",
+		"I'm not the atmos tech, but I'm great at laying pipe.",
 		"Could I have all access... to your heart?",
-		"Call me the doctor, because I'm here to inspect your johnson.",
+		"Call me the medical doctor, because I'm here to inspect your johnson.",
 		"Quick, get the defibrillator! I saw you and my heart stopped.",
 		"I'm not a changeling, but you make my proboscis extend.",
 		"I just can't get EI NATH of you.",
-		"You must be a nuke op, because you make my heart explode.",
-		"Roses are red / Botany is a farm / Not being my Valentine / causes human harm.",
+		"Roses are red / Intents are disarm / Not being my Valentine / causes human harm.",
```

Comment:
OOC / IC
This description needs changing
Intent's a game term

Reply:
not really

Reply:
like you can only refer to a "round" in a game context, but you can have in-character, an intent to push someone

Reply:
hu cares